Jellyfin.Plugin.KodiSyncQueue/EntryPoints/UserSyncNotification.cs

```diff
@@ -156,8 +155,20 @@ private void SaveUserChanges(List<MediaBrowser.Model.Dto.UserItemDataDto> dtos,
         {
             KodiSyncQueuePlugin.Instance.DbRepo.SetUserInfoSync(dtos, itemRefs, userId);
             List<string> ids = dtos.Select(s => s.ItemId).ToList();
+            var itemCount = ids.Count;
 
-            _logger.LogInformation("\"USERSYNC\" User {UserId}({Username}) posted {NumberOfUpdates} Updates: {Updates}", userId, userName, ids.Count, string.Join(",", ids.ToArray()));
+            if (itemCount > 0)
+            {
+                _logger.LogInformation(
+                    "User Data Sync: User {UserName} ({UserId}) posted {NumberOfUpdates} updates",
+                    userName,
+                    userId,
+                    itemCount);
+
+                _logger.LogDebug(
+                    "Updated items: {Updates}",
+                    ids.ToArray());
```

Comment:
```suggestion
                    ids);
```
this is already a list